### PR TITLE
fix(announcement): remove async migrations announcement from next release

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

We don't need to scream about this on 1.34.0

### CODE REVIEW

File: frontend/src/layout/navigation/TopBar/announcementLogic.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export enum AnnouncementType {
 
 // Switch to `false` if we're not showing a feature announcement. Hard-coded because the announcement needs to be manually updated anyways.
 const ShowNewFeatureAnnouncement = false
+const ShowAttentionRequiredBanner = false
```
Comment (macobo) [inline frontend/src/layout/navigation/TopBar/announcementLogic.ts:20]: `ShowAsyncMigrationRequiredBanner`?

Comment (yakkomajuri) [reply frontend/src/layout/navigation/TopBar/announcementLogic.ts:20]: Well I guess things are already a bit confusing here - we check `!asyncMigrationsOk` but return `AnnouncementType.AttentionRequired`

I went for the more general switch like above

**Review by macobo — APPROVED** [review]
All of this is super messy but ok.

**Comment by yakkomajuri** [comment]
It is indeed

**Review by tiina303 — APPROVED** [review]
agree with this being messy and also with us moving fast 🚢

**Pull request merged**